style.css/sixthsectionStyle.css

```diff
@@ -0,0 +1,27 @@
+.sixth-section {
+  text-align: center;
+}
+
+.sixth-section-header {
+  text-align: center;
+  background-color: #fa382f;
+  color: #ffffff;
+  display: inline-block;
+  font-weight: bold;
+  padding: 8px;
+  margin-top: 40px;
+}
+
+.skills {
+  padding: 0;
+  margin: 24px 40px;
+}
+
+.skills-item {
+  margin-top: 8px;
+}
+
+.skills3 {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
```

Comment:
Needs to be 3 columns.

Reply:
Already it's 3, isn't it?

Reply:
I mean 3 in one row. It's 3 but 2 in one row the other one at the bottom. You should change your grid-template-columns for that.

Reply:
Done.